content/news/2020-12-01-about-me.md

```diff
@@ -0,0 +1,74 @@
+
+<table>
+  <tr>
+     <td></td>
+     <td></td>
+  </tr>
+  <tr>
+    <td><img src="/images/news/hello.jpg" width=450 height=360></td>
+    <td><img src="/images/news/deborah_intern.jpg" width=214 height=390></td>
+  </tr>
+ </table>
+
+My name is Deborah Aanyu from Uganda. I am 23 years old. I come from a family of five children – I am the eldest. I have been in Catholic (single sex) school for most of my education, and it was during this time that I got to appreciate the beauty of God in the world, and in my life. I was in mixed school for only two years, up until University. As an adventurous person, I like to try out a lot of things; from art and painting, to singing, writing, sports (basketball), web development, among other things.
+
+I am generally optimistic and believe the good in people. I like to read and write. I like to dream, talk, listen and see the sunrise in the morning. I like to see the moonlight at night and feel the rain on my face. I like flowers. I like delicious food and comfortable shoes. I like warm clothes and a nice cup of tea. I also love babies, nature and laughter. I love food and listening to music.
+
+I have been studying a Computer Engineering degree at Makerere University, Kampala for the last four years, and I finished my final examinations just recently. So I’ve been really excited about getting done with school and  having the time to focus on what I really want to do which is software development and contributing to Open source!!
+
+I will be working on improving Mixxx DJ’s user manual for the Outreachy Internship. Mixxx DJ is an Open Source application for DJs that gives them the power to mix songs for free. Each day that I wake up, I am so grateful for the opportunities that I am given, and looking at how I got here, I can’t help but feel enthusiastic for the journey ahead, bumps and all! I’m here for all of it.
+
+I would love to travel and see the world someday, meet new people and make new friendships. There are many things that I would love to do, see, and experience. I am also working towards attaining seniority level with web development (particularly with JavaScript) and just the MERN stack in general. (Let’s give it one more year) I aspire to be successful at everything that I do so that I can share my journey and use my experience to inspire other people, especially women. I am passionate about women empowerment and would love to see more women leaders in the technology sector.
+
+## My top three core values
+
+### Honesty. Respect. Adventure
+
+#### Honesty
+
+As a young teen, I was very shy – bordering on fearful. I feared everybody including my own parents. I feared to do the wrong thing because I wanted to be in everybody’s good books. I needed the approval of my friends, family and everybody who cared about me. Because of this, I was telling lies all the time. I was turning into a diabolical liar.
+
+![why you lyin'?](https://media.giphy.com/media/50wNufvTEswz6/giphy.gif)
```

Comment:
We try to avoid embedding external resources, because it leaks user data and also might go offline without us noticing. Could you add it to the static directory and embed it from there? Like this: https://docs.getpelican.com/en/4.5.3/content.html#linking-to-static-files

Reply:
Okay. Let me do that.